synapse/storage/room.py

```diff
@@ -533,73 +533,92 @@ def block_room(self, room_id, user_id):
         )
         self.is_room_blocked.invalidate((room_id,))
 
+    def get_media_mxcs_in_room(self, room_id):
```

Comment:
please could you document the args and return types (sorry I know synapse is bad about this in general - but I'm trying to not let it get any worse)